Modules/posixmodule.c

```diff
@@ -12544,6 +12544,21 @@ static PyObject *
 os_mkfifo_impl(PyObject *module, path_t *path, int mode, int dir_fd)
 /*[clinic end generated code: output=ce41cfad0e68c940 input=73032e98a36e0e19]*/
 {
+#ifdef MS_WINDOWS
```

Comment:
You should raise an error if dir_fd is used.

You should document that the mode is ignored on Windows.